docs/learn-accounts.md

```diff
@@ -528,8 +528,89 @@ networks.
   <span class="address-changer-output" id="output" />
 </div>
 
-Alternatively, use [this handy subscan tool](https://polkadot.subscan.io/tools/ss58_transform) or
-[this simple address address convertor](https://polkadot-address-convertor.netlify.app/).
+Alternatively, use this [handy subscan tool](https://polkadot.subscan.io/tools/ss58_transform) or
+this [simple address convertor](https://polkadot-address-convertor.netlify.app/).
```

Comment:
We don't know this person or what they've done behind the scenes. To prevent phishing, let's remove this link and leave only Subscan's.